README.md

```diff
@@ -38,26 +38,26 @@ Create a manifest file from the `projects` in `config.py`:
 python .\generate_manifest.py
 ```
 
-This will generate a new `manifest.py` based on those projects and the pacakges wihtin them:
+This will output to stdout a manifest based on those projects and the packages within them:

```

Comment:
Can you add an example where you pipe it to a file as many people may wan tot do this e.g. `python .\generate_manifest.py > manifest.json`